scripts/configs/defaults/augmentations/asymmetric/cifar.yaml

```diff
@@ -0,0 +1,59 @@
+data:
+  augmentations:
+    - rrc:
+        enabled: True
+        crop_min_scale: 0.08
+        crop_max_scale: 1.0
+      color_jitter:
+        enabled: True
+        brightness: 0.4
+        contrast: 0.4
+        saturation: 0.2
+        hue: 0.1
+        prob: 0.8
+      grayscale:
+        enabled: True
+        prob: 0.2
+      gaussian_blur:
+        enabled: True
+        prob: 1.0
+      solarization:
+        enabled: False
```

Comment:
isnt it redundant to ave `enabled: False` and `prob: 0.0`? can't we jut remove enabled?

Reply:
@DonkeyShot21 hydra (and maybe omegaconf) has this weird constraint that if a config key is not defined, you cannot add it dynamically in code. So perhaps this is why

Reply:
@turian I think what Enrico meant was to replace `if cfg.solarization.enabled` with `if cfg.solarization.prob > 0`